.github/workflows/ci-validate-platforms.yml

```diff
@@ -57,6 +57,9 @@ jobs:
       - name: Install package dependencies
         run: yarn install --frozen-lockfile --ignore-scripts --network-timeout=360000 --prefer-offline
 
+      - name: Install Playwright
+        run: npx playwright install chromium --with-deps
```

Comment:
Why are you adding 'npx'?  https://github.com/npm/npx (now deprecated and part of NPM)

Reply:
I am not adding npx, npx comes with npm, that is why the individual package was deprecated. This is how playwright documentation suggests to install the browsers, see https://playwright.dev/docs/intro?msclkid=0d4f729aa6e411ecb1768b44af0fd72e#manually

Reply:
`yarn playwright install chromium --with-deps` should work the same way here

Reply:
I think this might be preference/opinion due to issue https://github.com/microsoft/fast/issues/5329, I'd say for future work not including `yarn` is inline with our goals outlined in the issue but I can see how using it could also be the case vis a vis consistency with current commands. I'll stick with future-forward position of using `npx`, unless someone has a big issue with it.

Reply:
We have a static version of Yarn 1.22.11 installed to `.yarn/releases/yarn-1.22.11.cjs` in the repository to avoid potential issues that can arise from developers using different versions of Yarn. This can also help with debugging, since we know that every user (including CI) is using the specific script from the repo and not an alternative package manager.

Fluent uses this baked-in Yarn behavior as well, and even goes further by running the [`use-yarn-please.js` script](https://github.com/microsoft/fluentui/blob/master/scripts/use-yarn-please.js) to prevent developers from using anything other than the locally-scoped version included in their monorepo.

Reply:
Understood that `yarn` is also viable, however I'm still not seeing any downside with my approach. It seems like we have different preferences, again unless someone has a big issue with it I'll move forward.

Reply:
I agree with @janechu here, barring some functional difference I'm not aware of. If we have a long-term goal to migrate away from Yarn, then I think it makes sense to favor npx.

Reply:
Greater consistency equals greater maintainability.  I'd recommend, sticking with Yarn in this case. If we want to fix it forward, then I'd recommend submitting a new PR to switch everything to NPX.  Otherwise, we're unnecessarily fragmenting our codebase with no guarantee future PRs would fix forward.

Reply:
I'm fine with switching to NPM, but I'm not 100% sold on using conflicting tools in CI when there's already precedence in our workflows for this exact situation: https://github.com/microsoft/fast/blob/082acb9ad25e3f27580f4f948b3446b276da65fc/.github/workflows/ci-validate-platforms.yml#L77

Reply:
npx does not conflict with yarn, it uses the same method of checking the node_modules folders for the installation before executing it.